common/src/main/java/tachyon/Constants.java

```diff
@@ -274,6 +274,8 @@
       "tachyon.user.file.master.client.threads";
   public static final String USER_LINEAGE_ENABLED = "tachyon.user.lineage.enabled";
 
+  public static final String USER_WAITCOMPLETED_POLL = "tachyon.user.waitcompleted.poll.millis";
```

Comment:
It is actually used in TachyonFileSystemUtils#waitCompleted, and I believe it is nice to let it be configurable by the user